client/my-sites/stats/overview.jsx

```diff
@@ -88,21 +88,22 @@ export default React.createClass( {
 			);
 
 			return overview;
-		}, this );
+		} );
 
 		return (
-			<Main wideLayout={ true }>
+			<Main wideLayout>
+				<QuerySites allSites />
```

Comment:
Careful, we're duplicating data in SitesList and and the redux store. Since we're doing this in the stats section means that it's probably being populated for everyone.

Should we revert this or maybe speed up SitesList conversion? cc @aduth @timmyc @youknowriad  @dmsnell